app/graphql/types/compute_attribute.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class ComputeAttribute < BaseObject
```

Comment:
This is rather useless right now if we don't have compute profiles, but I guess it can't hurt.